### ISSUE TITLE
2.0.0 Concept

### ISSUE THREAD

**Issue by SergeShkurko** [tars/tars-cli#82]
TODO:
- [x] Change modules import syntax from commonjs to es6
- [x] Add set project name (to package.json) answer
- [ ] Update packages
- [x] Remove Jade
- [x] Remove `re-init` command

Possibly:
- [ ] Add tests
- [ ] Transfer main functional from tars/tars.

**Comment by SergeShkurko** [comment]
SergeShkurko/tars-cli

**Comment by artem-malko** [comment]
Change modules import syntax from commonjs to es6 — не будет ли это медленнее для работы самого сборщика, старт будет дольше, так как код нужно транспилить. И не совсем понятен профит такой правки. 
Было бы лучше перенести все на TS и запускать через ts-node. Тогда и импорты будут, и типы, и вообще сказка.

**Comment by artem-malko** [comment]
С остальным согласен.

**Comment by fdsea** [comment]
Привет! Запускаю сборку tars dev и вылетает ошибка что пакет gulp-sass недоступен

**Comment by SergeShkurko** [comment]
@artem-malko нет, не будет. Код собирается при помощи babel и на выходе получаются обычные commonjs модули (https://github.com/SergeShkurko/tars-cli/blob/c94348180568e1449e1ab5b79d68f603472876da/package.json#L118)

**Comment by artem-malko** [comment]
@SergeShkurko а насколько замедлился запуск?

**Comment by SergeShkurko** [comment]
@artem-malko не более чем на миллисекунды [вот пример bin/tars.js](https://pastebin.com/VEuvSz9u)

**Comment by SergeShkurko** [comment]
По поводу TS - это конечно хоть и мой любимый язык, но tars слишком большой и возникнет несоизмеримо большое кол-во проблем при адаптации. Как вариант можно рассмотреть полный рефакторинг с оптимизацией и в дальнейшем адаптация к TS
